lib/locale-de-DE.json

```diff
@@ -0,0 +1,46 @@
+{
+  "n": {
+    ".gitignore": "Ignorierdatei (.gitignore)",
+    "repository": "Projektordner (repository)",
+    "repositories": "Projektordner (repositories)",
+    "branch": "Zweig (branch)",
+    "branches": "Zweige (branches)",
+    "fork": "Kopie (fork)",
+    "forks": "Kopien (forks)",
+    "request": "Anfrage (request)",
+    "pull request": "Änderungsanfrage (pull request)",
+    "pull requests": "Änderungsanfragen (pull requests)",
+    "commit": "Änderungssatz (commit)",
+    "commits": "Änderungssätze (commits)",
+    "upstream": "ursprüngliches Projekt (upstream)",
+    "feature": "Funktionalität (feature)",
+    "status": "Status",
+    "diff": "Unterschiede (diff)",
+    "readme": "Hilfedatei (readme)",
+    "collaborator": "Teammitglied (collaborator)",
+    "collaborators": "Teammitglieder (collaborators)",
+    "remote": "zentraler Projektordner (remote)",
+    "remotes": "zentrale Projektordner (remotes)",
+    "contributing": "mitmachen (contributing)",
+    "connections": "Verbindungen (connections)"
+  },
+  "v": {
+    "add": "hinzufügen (add)",
+    "fork": "kopieren (fork)",
+    "forked": "kopiert (forked)",
+    "clone": "Arbeitskopie erstellen (clone)",
+    "push": "hochladen (push)",
+    "close": "schließen (close)",
+    "pull": "herunterladen (pull)",
+    "commit": "Änderung bestätigen (commit)",
+    "merge": "zusammenführen (merge)",
+    "merged": "zusammengeführt (merged)",
+    "checkout": "wechseln (checkout)",
+    "stage": "vorbereiten (stage)"
```

Comment:
Would "vorhalten" or "zurückhalten" be a possible alternative? If i think about when to use the git command "git stash"  i wouldnt understand it as "prepare" but rather more as "keep back" or "hold up" the changes.